.github/workflows/legacy-checkpoints.yml

```diff
@@ -64,6 +64,18 @@ jobs:
         aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY_ID }}
         aws-region: us-east-1
 
+    - name: Install lite from source for PL
+      # Lite shall have pin version, so to keep development aligned we need to install lite from source not from pypi
```

Comment:
Why don't we work on fixing this always instead of just for this workflow? #15489